server/modules/rendering/html-image-prefetch/renderer.js

```diff
@@ -0,0 +1,23 @@
+const request = require('request-promise')
+
+const prefetch = async (element) => {
+  const url = element.attr(`src`)
+  const response = await request({
+    method: `GET`,
+    url,
+    resolveWithFullResponse: true
+  })
+  const contentType = response.headers[`content-type`]
+  const image = Buffer.from(response.body).toString('base64')
+  element.attr('src', `data:${contentType};base64,${image}`)
+  element.removeClass('prefetch-candidate');
+}
+
+module.exports = {
+  async init($) {
+    const promises = $('img.prefetch-candidate').map((_, element) => {
```

Comment:
To avoid confusion, don't use `_` as a variable name unless it's lodash.

Reply:
fixed